src/blogApp/errorHandlers.js

```diff
@@ -0,0 +1,58 @@
+export const dbError = (res, message = "Unable to connect to the database") => {
```

Comment:
For the error handlers, you can create a generic handler where you pass parameters rather than repeating yourself. **DRY**

Reply:
I added that error handler in the next PR